glean_parser/parser.py

```diff
@@ -62,7 +62,7 @@ def _load_file(
     `parser_config["allow_missing_files"]` is `True`.
     """
     try:
-        content = util.load_yaml_or_json(filepath, ordered_dict=True)
+        content = util.load_yaml_or_json(filepath)
```

Comment:
`ordered_dict` was removed because with the new yaml loader, OrderedDict is always used as the new dict subclass